sound/soc/sof/intel/hda.c

```diff
@@ -666,6 +670,8 @@ int hda_dsp_probe(struct snd_sof_dev *sdev)
 	iounmap(sdev->bar[HDA_DSP_BAR]);
 hdac_bus_unmap:
 	iounmap(bus->remap_addr);
+/* disable_pci: not currently used */
```

Comment:
comment intend looks wired here.

Reply:
it was intentional to mirrow what is done for other labels not used either.